go/vt/vttablet/tabletserver/vstreamer/rowstreamer.go

```diff
@@ -227,14 +229,19 @@ func (rs *rowStreamer) streamQuery(conn *snapshotConn, send func(*binlogdatapb.V
 		Fields:   rs.plan.fields(),
 		Pkfields: pkfields,
 		Gtid:     gtid,
+		Done:     false,
 	})
 	if err != nil {
 		return fmt.Errorf("stream send error: %v", err)
 	}
 
 	response := &binlogdatapb.VStreamRowsResponse{}
+	response.Done = false
```

Comment:
this will always get initialzed to false so you don't have to explicitly set it.